src/components/CategoryFilter.js

```diff
@@ -0,0 +1,33 @@
+import PropTypes from 'prop-types';
+
+const CategoryFilter = props => {
+  const categories = ['All', 'Action', 'Biography', 'History', 'Horror', 'Kids', 'Learning', 'Sci-Fi'];
+  const { handleFilter } = props;
+
+  const handleClick = event => {
+    const { value } = event.target;
+    // handleFilter(value);
+    // console.log(value);
+    if (value === 'All') {
+      handleFilter(null);
+    } else {
+      handleFilter(value);
+    }
+  };
+
+  return (
+    <label htmlFor="select">
+      Filter by category:
+      <select name="category" id="select" onChange={handleClick}>
+        <option>Select Category</option>
```

Comment:
**OPTIONAL**
We don't need this option element since 'All' should be our default category. Instead of using an option, we can use a header or some other element before declaring select element